symforce/codegen/backends/python/templates/util/util.jinja

```diff
@@ -120,6 +120,45 @@ def {{ function_name_and_args(spec) }}:
 
 {# ------------------------------------------------------------------------- #}
 
+{# Helper for expr_code. Checks that an input matrix called name has a valid shape,
+ # raises an IndexError if invalid, and reshapes to be a 2d ndarray.
+ #
+ # If use_numba=True, the IndexError message raised does not say what the size of the
+ # argument passed in was. This is because numba compatible code must have any error
+ # messages known as compile time. Similarly, the reshaping is always performed
+ # because numba comptibly code cannot reshape an array in a conditional.
```

Comment:
typo: compatible?

Reply:
Yeah, this one is a bit more embarrassing...